eduHub/src/test/kotlin/com/arsiu/eduhub/it/grcp/AssignmentGrpcServiceTest.kt

```diff
@@ -16,20 +16,27 @@ import io.grpc.testing.GrpcCleanupRule
 import net.devh.boot.grpc.client.inject.GrpcClient
 import org.junit.Rule
 import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation
+import org.junit.jupiter.api.Order
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestMethodOrder
 import org.junit.jupiter.api.extension.ExtendWith
 import org.springframework.boot.test.context.SpringBootTest
 import reactor.core.publisher.Flux
 import reactor.test.StepVerifier
+import java.util.concurrent.CopyOnWriteArrayList
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
 
 @SpringBootTest(
     properties = [
         "grpc.server.inProcessName=test",
-        "grpc.server.port=-1",
+        "grpc.server.port=9091",
         "grpc.client.inProcess.address=in-process:test"
     ]
 )
 @ExtendWith(TestContainers::class)
+@TestMethodOrder(OrderAnnotation::class)
```

Comment:
your test should be independent, refactor tests and remove order, please

Reply:
`+` just made it because I was having issue that delete test deletes entity when my stream is trying to emit him